template_library/views.py

```diff
@@ -0,0 +1,31 @@
+from allauth.account.views import ConfirmEmailView
+from allauth.socialaccount.providers.facebook.views import FacebookOAuth2Adapter
+from allauth.socialaccount.providers.google.views import GoogleOAuth2Adapter
+from django.shortcuts import redirect
+from rest_auth.registration.views import SocialLoginView
+
+
+class FacebookLogin(SocialLoginView):
+    adapter_class = FacebookOAuth2Adapter
+
+
+class GoogleLogin(SocialLoginView):
+    adapter_class = GoogleOAuth2Adapter
+
+
+class CustomConfirmEmailView(ConfirmEmailView):
+    """
+    Override default register view to
```

Comment:
the docstring looks incorrect, it mentions the register view but the view is inheriting `ConfirmEmailView`, also, the name `CustomConfirmEmailView` is not great, I'd prefer to have:

```python
from allauth.account.views import BaseConfirmEmailView


class ConfirmEmailView(BaseConfirmEmailView):
    ...
```